sys/dev/ath/if_ath_usb.c

```diff
@@ -503,7 +503,7 @@ ath_usb_detach(device_t self)
 	struct ath_softc *sc = &usc->sc_sc;
 
 	if (usc->sc_ath_attached)
-		ath_detach(sc);
+		//ath_detach(sc);
```

Comment:
`ath_detach` is called also by `ath_pci_detach`. I understand it's commented because dynamic linker has faulted here. Maybe adding if_ath.c file to Makefile woudl fix it? You can check that or create a new task for that

Reply:
I think making new task should be ok

Reply:
I made new task for detach